src/ui/reducers/network.ts

```diff
@@ -26,6 +28,15 @@ const update = (state: NetworkState = initialState(), action: NetworkAction): Ne
         events: [...action.payload.events, ...state.events],
         requests: [...action.payload.requests, ...state.requests],
       };
+    case "NEW_RESPONSE_BODY_PARTS":
+      console.log({ action, responsesValue: state.responses[action.payload.responseBodyParts.id] });
+      return {
+        ...state,
+        responses: {
+          ...state.responses,
+          [action.payload.responseBodyParts.id]: action.payload.responseBodyParts.parts,
+        },
```

Comment:
I'm somewhat worried we're going to end up crashing the user's tab by collecting all this data in the store, but maybe we can see what happens. Might be good to discard entries in here that haven't been used in a while.

Reply:
If we're only going to request at most 5MB with the change up above, we should be OK I think. We only load this when they go to the response tab, so they would have to go to a *lot* of response tabs to make a big memory dent.